### DISCUSSION TITLE
Daily 21/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#92] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Ayer hice prácticamente todo lo que dije en la anterior Daily, hoy sigo con Java y voy a tratar de meterle mano a android, como también realizar algún cuestionario más de ciber.

**Comment by NataliaAlvarezIspc** [comment]
Ayer revisé y completé el ieee, también subí el diagrama de clases a la carpeta de documentación. Estuve armando las subtareas que se desprenden de las tareas originales. Además practiquè inglés para la clase de la noche en la que participé. También estuve en la oficina virtual.
Hoy voy a empezar con la activity de login, me cuesta bastante pero voy a hacer lo posible por terminarla entre hoy y mañana. Por la tarde voy a participar de la clase de testing.

**Comment by Valentino399** [comment]
Antes de ayer estuve realizando cuestionarios de ciberseguridad y ayer estuve viendo videos en Youtube de android y hoy hice un cuestionario mas de cinerseguridas y tratare de aprender mas acerca de android
